addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -794,8 +794,8 @@ blockquote {
         background-image: none !important;
     }
     &:hover {
-        background-color: '';
-        background-image: '';
+        background-color: transparent;
+        background-image: none;
```

Comment:
I am not sure about this, your commit message is not enough for my small brain to understand why we need one side to be important and not the other. And even if it was, this really needs inline comments, someone seeing this has no chance to understand except by blaming (which we should never have too for this kind of question).

So my question, I guess, is, why not this?
```
.btn-outline-custom {
        background-color: transparent !important;
        background-image: none !important;
}
```
Although may be not entirely stable... but if this is the reason, there should a TODO for master as inline comment so that someone stumbling upon this rule knows what is going on :+1:

Reply:
Outline buttons are transparent by default but filled on hover.
It cannot be applied globally (hover + non-hover): the existing customized color/gradient stored in the element style is not `!important` and therefore does not appear on hover, and if it was `!important` then it would also appear in non-hover.
However, the non-hover part of the rule can be removed: `background-color: transparent;` is already defined on `.btn` and if no `background-image` is defined in the style - there is no such attribute to neutralize anyway.
I will verify in #75680 if the hover rule is needed - but this is unlikely because this applies to the custom style anyway and the override would only be required if the button was also primary or secondary.